authui/src/inline-preview.ts

```diff
@@ -89,6 +91,9 @@ export class InlinePreviewController extends Controller {
       outlet?.setValue(value);
     }
 
+    el.classList.remove("light", "dark", "auto");
+    el.setAttribute("class", customisationMessage.theme);
```

Comment:
If you look at authui/src/colorscheme.ts, you will see there is no `light` or `auto`. You have to study how light/dark works in Auth UI first. You never guess.